src/data/ACTIONS/layers/patch6.1.ts

```diff
@@ -56,5 +56,9 @@ export const patch610: Layer<ActionRoot> = {
 			icon: 'https://xivapi.com/i/002000/002649.png',
 			cooldown: 1000,
 		},
+
+		PET_SEARING_LIGHT: {
+			statusesApplied: [],
+		},
```

Comment:
Please keep changes for a single job together, this has WHM stuff in between 2 chunks of SMN changes.

Reply:
hmm, not sure why I put it there. moved.